eZ/Publish/Core/settings/storage_engines/legacy/bookmark.yml

```diff
@@ -0,0 +1,31 @@
+parameters:
+    ezpublish.persistence.legacy.bookmark.gateway.inner.class: eZ\Publish\Core\Persistence\Legacy\Bookmark\Gateway\DoctrineDatabase
+    ezpublish.persistence.legacy.bookmark.gateway.exception_conversion.class: eZ\Publish\Core\Persistence\Legacy\Bookmark\Gateway\ExceptionConversion
+    ezpublish.persistence.legacy.bookmark.mapper.class: eZ\Publish\Core\Persistence\Legacy\Bookmark\Mapper
+    ezpublish.spi.persistence.legacy.bookmark.handler.class: eZ\Publish\Core\Persistence\Legacy\Bookmark\Handler
```

Comment:
Please drop class parameters as Symfony dropped them a long time ago for `3.0`.
TBH new files could use class names for service names, maybe when service is used outside of this file it should just have an alias.